client/src/messages.ts

```diff
@@ -304,8 +315,13 @@ export interface PickFrameDetails {
     parameters: PickFrameParams,
 }
 
+export interface RadialFourierDetails {
+    type: AnalysisTypes.RADIAL_FOURIER,
+    parameters: RadialFourierParams,
+}
+
 export type AnalysisParameters = MaskDefRing | MaskDefDisk | CenterOfMassParams | PointDef | PickFrameParams;
```

Comment:
Hmm, we may need `RadialFourierParams` as part of `AnalysisParameters`, too. Not sure why it is not complaining, because the `updateParameters` action uses this type...